<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>archaeoLinesDialog</class>
 <widget class="QWidget" name="archaeoLinesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArchaeoLines Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>530</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>ArchaeoLines Plug-in Configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>ArchaeoLines</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="switchToWindowedModeLabel">
         <property name="text">
          <string>To change colors you may have to leave fullscreen mode!</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="equinoxCheckBox">
           <property name="text">
            <string>Equinox line (Equator)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="equinoxColorToolButton">
           <property name="text">
            <string notr="true">EquinoxColor...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="solsticesCheckBox">
           <property name="text">
            <string>Solstice Lines</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="solsticesColorToolButton">
           <property name="text">
            <string notr="true">SolsticeColor...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="crossquarterCheckBox">
           <property name="text">
            <string>Crossquarter Lines</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="crossquarterColorToolButton">
           <property name="toolTip">
            <string>Change color</string>
           </property>
           <property name="text">
            <string notr="true">CrossquarterColor...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="majorStandstillCheckBox">
           <property name="text">
            <string>Major Lunar Standstill lines</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="majorStandstillColorToolButton">
           <property name="text">
            <string notr="true">majorLunarStandstillColor...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="minorStandstillCheckBox">
           <property name="text">
            <string>Minor Lunar Standstill lines</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QToolButton" name="minorStandstillColorToolButton">
           <property name="text">
            <string notr="true">minorLunarStandstillColor...</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="zenithPassageCheckBox">
           <property name="text">
            <string>Declination of Zenith passage</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QToolButton" name="zenithPassageColorToolButton">
           <property name="text">
            <string notr="true">zenithPassageColor...</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="nadirPassageCheckBox">
           <property name="text">
            <string>Declination of Nadir passage</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QToolButton" name="nadirPassageColorToolButton">
           <property name="text">
            <string notr="true">NadirPassageColor...</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="currentSunCheckBox">
           <property name="text">
            <string>Current Sun</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QToolButton" name="currentSunColorToolButton">
           <property name="text">
            <string notr="true">SunColor...</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="currentMoonCheckBox">
           <property name="text">
            <string>Current Moon</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QToolButton" name="currentMoonColorToolButton">
           <property name="text">
            <string notr="true">MoonColor...</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QComboBox" name="currentPlanetComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Current Planet: None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Planet: Mercury</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Planet: Venus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Planet: Mars</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Planet: Jupiter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current Planet: Saturn</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QToolButton" name="currentPlanetColorToolButton">
           <property name="text">
            <string notr="true">PlanetColor...</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="selectedObjectCheckBox">
           <property name="text">
            <string>Selected Object</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QToolButton" name="selectedObjectColorToolButton">
           <property name="text">
            <string notr="true">SelectedObjectColor...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="formatDisplayBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Format</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="useDmsFormatCheckBox">
            <property name="text">
             <string>Use dms format for angles</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="restoreDefaultsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Restore default settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string comment="tab in plugin windows">About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="aboutTextBrowser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
